### PR TITLE
[docs] Edit survey links

### PR DESCRIPTION
**Pull request by richardliaw**
## Why are these changes needed?
Add distributed SGD survey, remove Tune survey.

## Checks

- [ ] I've run `scripts/format.sh` to lint the changes in this PR.
- [ ] I've included any doc changes needed for https://ray.readthedocs.io/en/latest/.
- [ ] I've made sure the tests are passing. Note that there might be a few flaky tests, see the recent failure rates at https://ray-travis-tracker.herokuapp.com/.

### CODE REVIEW

File: doc/source/raysgd/raysgd_pytorch.rst [hunk 1]
```diff
@@ -3,6 +3,8 @@ RaySGD Pytorch
 
 .. warning:: This is still an experimental API and is subject to change in the near future.
 
+.. tip:: Help take a 1 minute `User Survey for the Ray Distributed Training library <https://forms.gle/26EMwdahdgm7Lscy9>`_!
```
Comment (deanwampler) [inline doc/source/raysgd/raysgd_pytorch.rst]: How about "tip:: Help us make Ray distributed Training better; take this 1 minute user survey..."

File: doc/source/raysgd/raysgd_tensorflow.rst [hunk 2]
```diff
@@ -1,6 +1,10 @@
 RaySGD TensorFlow
 =================
 
+.. warning:: This is still an experimental API and is subject to change in the near future.
+
+.. tip:: Help take a 1 minute `User Survey for the Ray Distributed Training library <https://forms.gle/26EMwdahdgm7Lscy9>`_!
```
Comment (deanwampler) [inline doc/source/raysgd/raysgd_tensorflow.rst]: Same comment as for the pytorch page.

File: doc/source/tune-tutorial.rst [hunk 3]
```diff
@@ -1,8 +1,6 @@
 Tune Walkthrough
 ================
 
-.. tip:: Help make Tune better by taking our 3 minute `Ray Tune User Survey <https://forms.gle/7u5eH1avbTfpZ3dE6>`_!
```
Comment (deanwampler) [inline doc/source/tune-tutorial.rst]: I think this older wording is better.

**Review by deanwampler — DISMISSED** [review]
Minor wording changes suggested. Otherwise, looks good.

**Review by simon-mo — APPROVED** [review]
lgtm as long as Dean's comments are addressed.

**Pull request merged**